src/main/java/com/training/web/pages/flipkart/CartPage.java

```diff
@@ -1,37 +1,34 @@
 package com.training.web.pages.flipkart;
 
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
 import com.training.basepages.FlipkartBasePage;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import java.util.ArrayList;
-import java.util.List;
+import static com.codeborne.selenide.Condition.appear;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
 
 public class CartPage extends FlipkartBasePage {
 
-    private By totalPrice = By.xpath("//*[contains(@class,'_3X7Jj1')]");
-    private By name = By.xpath("//a[contains(@class,'gBNbID')]");
-    private By placeOrderButton = By.xpath("//*[text()='Place Order']");
-
-    public CartPage(WebDriver driver) {
-        super(driver);
-    }
+    private SelenideElement totalPrice = $x("//*[contains(@class,'_3X7Jj1')]");
+    private ElementsCollection name = $$x("//a[contains(@class,'gBNbID')]");
+    private SelenideElement placeOrderButton = $x("//*[text()='Place Order']");
```

Comment:
Lets define this in a string rather than SelenideElement, and then use it in your methods